diff --git a/site/src/Helper/Cwmmedia.php b/site/src/Helper/Cwmmedia.php
index 21415462f..9faa528ac 100644
--- a/site/src/Helper/Cwmmedia.php
+++ b/site/src/Helper/Cwmmedia.php
@@ -198,18 +198,6 @@ class Cwmmedia
         if ($link_type < 2 && $params->get('show_filesize') > 0) {
             $file_size = (int)$media->params->get('size', 0);
 
-            if (!$file_size && $link_type !== 0) {
-                $file_size = Cwmhelper::getRemoteFileSize(
-                    Cwmhelper::mediaBuildUrl(
-                        $media->sparams->get('path'),
-                        $media->params->get('filename'),
-                        $params,
-                        true
-                    )
-                );
-                Cwmhelper::setFileSize($media->id, $file_size);
-            }
-
             if ($file_size > 1) {
                 $file_size = $this->convertFileSize($file_size, $params, $media);
                 $filesize  = '<span class="JBSMFilesize text-body-secondary small ms-1" style="white-space:nowrap;">' .
@@ -666,18 +654,6 @@ class Cwmmedia
 
         $file_size = (int) $media->params->get('size', '0');
 
-        if ($file_size === 0) {
-            $file_size = Cwmhelper::getRemoteFileSize(
-                Cwmhelper::mediaBuildUrl(
-                    $media->sparams->get('path'),
-                    $params->get('filename'),
-                    $params,
-                    true
-                )
-            );
-            Cwmhelper::setFileSize($media->id, $file_size);
-        }
-
         if ($file_size > 1) {
             $file_size = $this->convertFileSize($file_size, $params, $media);
         }
@@ -972,12 +948,7 @@ class Cwmmedia
                     true
                 );
 
-                if ($media->params->get('size') === '0') {
-                    $size = Cwmhelper::getRemoteFileSize($url);
-                    Cwmhelper::setFileSize($media->id, $size);
-                } else {
-                    $size = $media->params->get('size');
-                }
+                $size = $media->params->get('size', 0);
 
                 $url = Cwmhelper::removeHttp($url);
 
